src/Componet/Home.jsx

```diff
@@ -1,10 +1,51 @@
 // import Eu from "../imagrnd/Eu.jpg"
+import { useState } from "react"
 import home from "../Componet/home.css"
 
 export default function Home() {
+    const [cor,setcor]= useState(1);
```

Comment:
mantenha o padrão no código. use somente inglês

```suggestion
    const [color,setColor]= useState(1);
```

outra coisa, pq uma cor é um número? refatore pra essa variável ter valores que façam sentido pra uma cor, como "red", "primary", "secondary", "#111"..